src/common/settings/whitelabel/CustomColorsEditorViewModel.ts

```diff
@@ -47,39 +50,16 @@ export class CustomColorsEditorViewModel {
 		this._entityClient = entityClient
 		this._loginController = loginController
 		this.builtTheme = stream()
-		const baseThemeId = themeCustomizations.base ?? "light"
 
+		const baseThemeId = themeCustomizations.base ?? "light"
 		const accentColor = themeCustomizations.primary ?? this._themeController.getDefaultTheme().primary
-
-		this.changeBaseTheme(baseThemeId)
-		this.changeAccentColor(accentColor)
+		this.changeTheme({ accentColor, baseThemeId })
```

Comment:
if this is `async` should we just do it in `init()`? And then we can get rid of `inited` since `init()` can await